jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1441,7 +1711,137 @@ function Get-ProtocolTypeAssociation {
     return $manifestList
 }
 ##### END MIT License #####
+$Script:ErrorMessage = $null
```

Comment:
This variable is populated when `Write-AdmuErrorMessage` is called then gets passed to the end of `Start-Migration` when there is a failure. Since it needs to be populated outside the `Start-Migration` function, I think this is the right place for it to keep it a `$script `variable so it can be accessed/populated by two different functions, unless I am missing anything?